theano/configdefaults.py

```diff
@@ -242,6 +242,16 @@ def filter_nvcc_flags(s):
              FloatParam(0),
              in_c_key=False)
 
+AddConfigVar('gpuarray.sched',
+             """The sched parameter passed for context creation to pygpu.
+                With CUDA, using "multi" mean using the parameter
+                cudaDeviceScheduleYield. This is useful to lower the
+                CPU overhead when waiting for GPU. One user found that it
+                speed up its other process that was doing data augmentation.
+             """,
+             EnumStr("default", "multi", "single"),
+             in_c_key=True)
```

Comment:
I updated the PR.

On Wed, Aug 3, 2016 at 5:05 PM, abergeron <notifications@github.com> wrote:

> In theano/configdefaults.py
> <https://github.com/Theano/Theano/pull/4821#discussion_r73420792>:
>
> > @@ -242,6 +242,16 @@ def filter_nvcc_flags(s):
> >               FloatParam(0),
> >               in_c_key=False)
> >
> > +AddConfigVar('gpuarray.sched',
> > +             """The sched parameter passed for context creation to pygpu.
> > +                With CUDA, using "multi" mean using the parameter
> > +                cudaDeviceScheduleYield. This is useful to lower the
> > +                CPU overhead when waiting for GPU. One user found that it
> > +                speed up its other process that was doing data augmentation.
> > +             """,
> > +             EnumStr("default", "multi", "single"),
> > +             in_c_key=True)
>
> This should not be in the C key, it doesn't change any code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4821/files/c02fc30cefd27ca1e510e17fd28429c1591a64a7#r73420792>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC--i2QYD4WC5UqiVugCCTcZxmeOxZks5qcQKkgaJpZM4JcEcQ>
> .
>